src/translations/de/translations.json

```
@@ -355,6 +355,7 @@
     "transaction": "Gebühr"
   },
   "paymentChoicePage": {
+    "saved": "Salvati",
```

```suggestion
    "saved": "Gespeicherte",
```

use same value used for IO app as translation